GithubRepo/Data/Services/LocalGithubFetcher.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  LocalGithubFetcher.swift
+//  GithubRepo
+//
+//  Created by Marcos Alves on 15/10/21.
+//
+
+import Foundation
+
+struct LocalGithubFetcher: GithubFetcher {
```

Comment:
This class name looks like you are fetching from some local database. I would prefer to change to `MockGithubFetcher`. Also, this kind of objects should be only in your test target for tests purposes.